### PR TITLE
Done: HW 07 in homework_07.py file. For task 7-10 data was taken from…

### PR DESCRIPTION
**Pull request by VeronikaDi**
Hello
HW 07 in homework_07.py file. For task 7-10 data was taken from homework_04.py
I'm not sure that I did tasks 7-10 correct. 
I tried to document functions 🤷‍♀️
Thank you

### CODE REVIEW

File: lesson_07/homework_07.py [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+# task 5
+"""  Написати функцію, яка приймає список слів та повертає найдовше слово у списку.
+"""
+
+
+def find_longest_word(words):
+    if not words:
+        return ""
+    long_word = max(words, key=len)
```
Comment (DartBrovsky) [inline lesson_07/homework_07.py:76]: It is no need to create separate local variable. It is enough to return max(arg) value.

File: lesson_07/homework_07.py [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+adwentures_of_tom_sawer_sentences = split_into_sentences(adwentures_of_tom_sawer)
+for sentence in adwentures_of_tom_sawer_sentences:
+    print(f"\n"
+          f"{sentence}")
+
+
+# 10 task = function
+
+
+def starts_with_by_the_time(sentences):
```
Comment (DartBrovsky) [inline lesson_07/homework_07.py:193]: If you create func that returns bool value - it`s name should start with is_starts_with_by_the_time()

**Review by DartBrovsky — APPROVED** [review]
I approve your solution. You age good in this homework. 
I leave small comments - take a look into them.
Thanks.

File: lesson_07/homework_07.py (hunk 1 above)
Comment (VeronikaDi) [reply lesson_07/homework_07.py:76]: Thank you for marks and explanations
will keep them in mind 🙂

**Pull request merged**